engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/ActivityTest.java

```diff
@@ -78,7 +78,7 @@ public final class ActivityTest {
                       .zeebeInput("https://github.com/{{orgId}}/{{repoId}}", "urlStatic")
                       .zeebeInputExpression(
                           "\"https://github.com/{{orgId}}/{{repoId}}\"", "urlExpression")
-                      .zeebeInput("My Name is &#34;Zeebe&#34;, nice to meet you", "quotesStatic")
```

Comment:
:wrench: Let's keep both test cases: `\"` and `&#34;`. The Modeler uses `&#34;` for double quotes.